### PR TITLE
Fix typos and remove stdlib dependencies

### PR DESCRIPTION
**Pull request by yukuku**
Removal of `kotlin-stdlib-jdk8` -related mentions, since stdlib is now automatically included.

Typos:
* KotlinComiple -> KotlinCompile
* `&args` -> `*args`

### CODE REVIEW

File: docs/topics/jvm/jvm-create-project-with-spring-boot.md [hunk 1]
```diff
@@ -153,7 +151,7 @@ fun main(args: Array<String>) {
    </def>
    <def title="Program entry point – main()">
       <p>The <a href="basic-syntax.md#program-entry-point"><code>main()</code></a> function is the entry point to the application.</p>
-      <p>It is declared as a <a href="functions.md#function-scope">top-level function</a> outside the <code>DemoApplication</code> class. The <code>main()</code> function invokes the Spring's <code>runApplication(&amp;args)</code> function to start the application with the Spring Framework.</p>
+      <p>It is declared as a <a href="functions.md#function-scope">top-level function</a> outside the <code>DemoApplication</code> class. The <code>main()</code> function invokes the Spring's <code>runApplication(*args)</code> function to start the application with the Spring Framework.</p>
```
Comment (koshachy) [inline docs/topics/jvm/jvm-create-project-with-spring-boot.md:154]:
```suggestion
      <p>It is declared as a <a href="functions.md#function-scope">top-level function</a> outside the <code>DemoApplication</code> class. The <code>main()</code> function invokes the Spring's <code>runApplication(&amp;args)</code> function to start the application with the Spring Framework.</p>
```

Comment (koshachy) [reply docs/topics/jvm/jvm-create-project-with-spring-boot.md:154]: That is an escaping of the asterisk character.

**Comment by koshachy** [comment]
Thanks for your contribution!
I've rolled back one thing, but overall this PR looks good!

File: docs/topics/jvm/jvm-create-project-with-spring-boot.md (hunk 1 above)
Comment (koshachy) [reply docs/topics/jvm/jvm-create-project-with-spring-boot.md]: Oh, I see.
Sorry, I'll create a separate commit with the fix.
It seems that I need a vacation after 1.9.0 release :D

Comment (yukuku) [reply docs/topics/jvm/jvm-create-project-with-spring-boot.md]: That must be tough. Get a nice rest and congratulations for the 1.9.0 release!

**Pull request merged**